src/Sulu/Bundle/ContactBundle/Resources/public/js/components/contacts/list/decorators/contact-view.js

```diff
@@ -107,7 +114,7 @@ define(function() {
          */
         bindGeneralDomEvents: function() {
             if (this.options.unselectOnBackgroundClick) {
-                this.sandbox.dom.on('.grid', 'click.contact.list', function() {
+                this.sandbox.dom.on('.page', 'click.contact.list', function() {
```

Comment:
@nnatter please use body to prevent possible side-effects when renaming the page-container